src/catalog_xliff.h

```diff
@@ -101,9 +102,6 @@ class XLIFFCatalog : public Catalog
 
     ValidationResults Validate(bool wasJustLoaded) override;
 
-    Language GetLanguage() const override { return m_language; }
-    void SetLanguage(Language lang) override { m_language = lang; }
-
     // FIXME: PO specific
```

Comment:
Addressed 0dcedfad2f58e975f240a73e25e12f335d99b23d and as side effect :) recalled why I changed what you mentioned in https://github.com/vslavik/poedit/pull/633#discussion_r433356649 - it's just to get rid redundant (as it seemed to me) stuff which keeps language in separate https://github.com/vslavik/poedit/blob/7a0e93fff167ae5aa786d7b74f7e16b78464c88c/src/catalog_xliff.h#L121 while there is https://github.com/vslavik/poedit/blob/7a0e93fff167ae5aa786d7b74f7e16b78464c88c/src/catalog.h#L384 in the base class already what confused at least me . So now I recalled there are 99% that no side effects or bugs/problem back when reverted back with this commit. So I am leaving completely up to you to decide whether to keep this commit so it it will revert back that `XLIFFCatalog::m_language` related stuff or to drop it if you still find sense in my intention/explanation/motivation